backend/src/kitconcept/intranet/distributions/intranet/content/content/195d96ca9952493cbffa69cda040b8b3/data.json

```diff
@@ -27,7 +27,7 @@
   "modified": "2025-07-25T15:25:07+00:00",
   "nav_title": null,
   "preview_caption_link": null,
-  "remoteUrl": "/features/content-types",
+  "remoteUrl": "http://nohost/Plone/features/content-types",
```

Comment:
@ericof I did not change anything on the example content. I did not even start the instance. However, I see a diff here. I am guessing this is a bug in exportimport? I am just curious here. cc @sneridagh @iRohitSingh

Reply:
@tisto @ericof plone.exportimport uses plone.restapi for (de)serialization. For an internal link like this, the URL gets converted into a UID-based URL internally, and then converted back to an absolute_url during serialization. When running the export script there's no real request so we get http://nohost/Plone as the absolute_url.

I guess we have to do something in plone.export to remove the portal url after getting the serialization from plone.restapi. Probably we are already doing this for other fields and just missed remoteUrl, since we aren't seeing the problem elsewhere.